packages/amplify_datastore/example/lib/Post.dart

```diff
@@ -105,4 +105,9 @@ class PostType extends ModelType<Post> {
   Post fromJson(Map<String, dynamic> jsonData) {
     return Post.fromJson(jsonData);
   }
+
+  @override 
+  Map<String, dynamic> toJson() {
+    return toJson();
+  }

```

Comment:
What's the significance of this? Looks like never ending, destroy the world kind of statement?

Reply:
Fixed.

Reply:
We can remove this, we only need toJson on Model right?